modules/appeals_api/app/models/appeals_api/evidence_submission.rb

```diff
@@ -6,7 +6,7 @@ module AppealsApi
   class EvidenceSubmission < ApplicationRecord
     include SetGuid
     self.ignored_columns = ['status'] # Temporary until migrations have run
-    belongs_to :supportable, polymorphic: true, optional: true
```

Comment:
so this is no longer optional? Is that intended?

Reply:
Yes. No longer optional.